### PR TITLE
Reorganized the bitnami base to support multiple namespaces

### PR DESCRIPTION
**Pull request by catalin-ilea**
shared dir contains kustomize bases that can be reused across namespaces config contains configuration files pass to the bitnami helm charts
   - currently contains the configuration for a tls cluster Makefile contains targets for each of the namespaces we want to create a base for
   - executing each target generates a kustomize base that is specific to each namespace which can be readily deployed

Pinned the bitnami chart to 22.1.1 and regenerated

### CODE REVIEW

File: bitnami/pubsub/upstream/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - kafka.yaml
+
+patches:
```
Comment (george-angel) [inline bitnami/pubsub/upstream/kustomization.yaml:7]: These don't feel "logging specific"

File: bitnami/gen-yaml/gen [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+mkdir -p ${NS}/upstream
+
+# Fetch from helm repo: https://github.com/bitnami/charts/tree/main/bitnami/kafka#installing-the-chart
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm template ${BITNAMI_KAFKA_RELEASE} bitnami/kafka --version ${BITNAMI_KAFKA_RELEASE} -f ${VALUES} --set fullnameOverride=${NAME} --set nameOverride=${NAME} --set kraft.clusterId=${KRAFT_CLUSTERID} --set auth.tls.existingSecrets="{${NAME}-0-tls,${NAME}-1-tls,${NAME}-2-tls}" --namespace ${NS} > ${NS}/upstream/kafka.yaml
+
+cat gen-yaml/clean-upstream-overlay.yaml > ${NS}/upstream/kustomization.yaml
+
+if [ ! -f ${NS}/kustomization.yaml ]; then
+cat gen-yaml/configure-logging-overlay.yaml > ${NS}/kustomization.yaml
```
Comment (sbuliarca) [inline bitnami/gen-yaml/gen:12]: I think we're using configure-logging-overlay.yaml more like the initial "final" kustomization file. I think we should rename it to something like "namespace-kustomization-template.yaml" or "instance-kustomization-stub.yaml".

File: bitnami/pubsub/upstream/kustomization.yaml (hunk 1 above)
Comment (catalin-ilea) [reply bitnami/pubsub/upstream/kustomization.yaml:7]: I agree it's not logging specific, template used for this is `clean-upstream-overlay.yaml`

File: bitnami/gen-yaml/gen (hunk 2 above)
Comment (catalin-ilea) [reply bitnami/gen-yaml/gen:12]: Yes, not the best name - how about `namespace-overlay.yaml` to keep it consistent with `clean-upstream-overlay.yaml`?

Comment (catalin-ilea) [reply bitnami/gen-yaml/gen]: renamed

**Review by sbuliarca — APPROVED** [review]

**Review by george-angel — APPROVED** [review]

**Pull request merged**